plugins/mayastor.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: mayastor
+spec:
+  version: v2.4.0
+  homepage: https://github.com/openebs/mayastor-extensions
```

Comment:
nit: this URL could be set to https://mayastor.gitbook.io/ or https://github.com/openebs/mayastor